src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -83,6 +83,13 @@ The .NET Foundation licenses this file to you under the MIT license.
       <SharedLibraryInstallName Condition="'$(SharedLibraryInstallName)' == '' and '$(NativeLib)' == 'Shared'">@rpath/$(TargetName)$(NativeBinaryExt)</SharedLibraryInstallName>
     </PropertyGroup>
 
+    <PropertyGroup Condition="'$(_targetOS)' == 'osx'">
+      <AppleMinOSVersion Condition="'$(AppleMinOSVersion)' == ''">11.0</AppleMinOSVersion>
+      <CrossCompileArch Condition="'$(CrossCompileArch)' == '' and $(RuntimeIdentifier.EndsWith('-x64'))">x86_64</CrossCompileArch>
+      <CrossCompileArch Condition="'$(CrossCompileArch)' == '' and $(RuntimeIdentifier.EndsWith('-arm64'))">arm64</CrossCompileArch>
```

Comment:
Not really happy about the `CrossCompile` in the name but introducing yet another property would probably be worse.

Reply:
This could be just:

```xml
    <PropertyGroup Condition="'$(_targetOS)' == 'osx'">
      <AppleMinOSVersion Condition="'$(AppleMinOSVersion)' == ''">11.0</AppleMinOSVersion>
      <TargetTriple Condition="'$(_targetArchitecture)' == ''x64">x86_64-apple-macos$(AppleMinOSVersion)</TargetTriple>
      <TargetTriple Condition="'$(_targetArchitecture)' == ''arm64">arm64-apple-macos$(AppleMinOSVersion)</TargetTriple>
    </PropertyGroup>
```